.ktlintignore

```diff
@@ -0,0 +1,7 @@
+// disabled ("./mvnw clean verify" fails with "Internal Error")
```

Comment:
This file is kind of goofy, but I liked the idea of allowing comments so that we can explain why a rule is disabled